### ISSUE TITLE
Branch clean up

### ISSUE THREAD

**Issue by lachlandeer** [tilburgsciencehub/website#601]
### Contact Details

lachlan.deer@gmail.com

### Shoot!

We currently have 63(!) branches on the main repo. 

Should trim back all branches where features have been merged to make this easier to navigate

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lachlandeer** [comment]
Done this.

Back to 4. 

Need a system to name branches , that will inherit from #605

Once a branch is merged into `master` we should delete it.

Consider doing it [automatically](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches), or do we think should wait 30 days? @shrabasteebanerjee

**Comment by shrabasteebanerjee** [comment]
i think automatically saves us some work right? and if changes have been merged we don't need to keep a record of the branch!

**Comment by lachlandeer** [comment]
I was worried if we were to merge something that lead to a side effect we didnt like... but if we move to a workflow that looks more like [this](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) which is where I would want to head towards we'd likely be OK
